src/RSCPL.xml

```diff
@@ -0,0 +1,473 @@
+<SPDX name="Ricoh Source Code Public License" identifier="RSCPL" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/RSCPL</url>
```

Comment:
Possibly link to [here][1] or similar to show the upstream location, using the Internet Archive because risource.org seems to have been abandoned.

[1]: http://wayback.archive.org/web/20060715140826/http://www.risource.org/RPL/RPL-1.0A.shtml